config/config.go

```diff
@@ -45,6 +45,8 @@ const (
 	AuthStrategyLoginIssuer     = "kiali-login"
 
 	// These constants are used for external services auth (Prometheus, Grafana ...) ; not for Kiali auth
+	// If you need to add more, make sure to update the schema of the operator crd:
+	//   $/operator/deploy/crd.yaml, the deployment.external_services property
```

Comment:
We need to remind people the CRDs are all over the place - it isn't just one yaml file. I do not know the best way to do this. But, for example, a new CRD requires us to add a new CSV in the manifests (we have 3 different manifest locations today, [upstream](https://github.com/kiali/kiali/tree/v1.9/operator/manifests/kiali-upstream), [community](https://github.com/kiali/kiali/tree/v1.9/operator/manifests/kiali-community), and [ossm](https://github.com/kiali/kiali/tree/v1.9/operator/manifests/kiali-ossm)) and in there we need to add a new CRD. For example: https://github.com/kiali/kiali/blob/v1.9/operator/manifests/kiali-community/1.7.0/kiali.crd.yaml

Maybe this is good enough:

```suggestion
	//   (i.e. $/operator/deploy/crd.yaml and the CRDs defined in the OLM operator manifests), the deployment.external_services property
```

Reply:
OK, I used your suggestion